virt_lightning/shell.py

```diff
@@ -266,24 +259,38 @@ def main():
         default="default",
         help="change the name of the context (default: %(default)s)",
     )
+    parser.add_argument(
+        "--config",
+        help="path to configuration file",
+        required=False,
```

Comment:
I guess we can safely set a default value here. E.g: ~/.config/virt-lightning/config.ini

Reply:
I this case need remove DEFAULT_* dict settings and delivery configuration to `$HOME` or build while running an app. Also need provide version of config file to rebuilt on required new properties if we'll use config file at home directory.

Reply:
Why? If there is no configuration file , or if a key is missing, we still need to define a default value somewhere.

Reply:
So, we can set default file configuration. Does user put file to `~/.config/virt-lightning/config.ini`?

Reply:
If the default file does not exist, we ignore it.

Bonus point for later: If the user point --config on something that does not exist, we should raise an exception.